web/templates/user_group_info_popover.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="group-info-popover">
     <div class="popover-inner">
-        <div class="popover-content">
+        <div class="tippy-content">
```

Comment:
Is `tippy-content` a class tippy itself sets? If so, I think the correct thing to do is actually delete the div; and also probably `popover-inner` is also unnecessary.

Reply:
563f8aba1e4bd1dcf19e6c9f3e342ddee6f8d30f is an example popover migration commit that seems to confirm we don't need those extra wrapper divs.